Wrapping/Generators/Python/itk/support/itkLazy.py

```diff
@@ -51,13 +57,23 @@ def __init__(self, name, lazy_attributes):
     def __getattribute__(self, attr):
         value = types.ModuleType.__getattribute__(self, attr)
         if value is not_loaded:
-            module = self.__belong_lazy_attributes[attr]
-            namespace = {}
-            itkBase.itk_load_swig_module(module, namespace)
-            self.loaded_lazy_modules.add(module)
-            for k, v in namespace.items():
-                setattr(self, k, v)
-            value = namespace[attr]
+            with self._module_lock:  # All but one thread will block here.
+                if (
+                    value is not_loaded
+                ):  # Only the first thread needs to run this code, all other blocked threads skip
```

Comment:
This comment can be on a separate line, it does not have to be on the same line as the `if` statement. I think that forced the `if` to break into 3 lines.